modules/representation_management/spec/support/rswag_config.rb

```diff
@@ -43,7 +43,9 @@ def schemas
       ErrorModel: error_model,
       Errors: errors,
       Error: error,
-      PowerOfAttorneyResponse: power_of_attorney_response
+      PowerOfAttorneyResponse: power_of_attorney_response,
+      veteran_service_representative: veteran_service_representative_schema,
```

Comment:
It looks like there's a mix of case types here. Just checking, should the following be snakecase or camelcase like the keys above?
- `veteran_service_representative`
- `veteran_service_organization`

Reply:
I don't know if the naming conventions matter but I'll make these match the others.

Reply:
They're updated and reflected in the new screenshot above.